Project.toml

```diff
@@ -21,6 +21,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Setfield = "efcf1570-3423-57d1-acb7-fd33fddbac46"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
```

Comment:
Why do we need this package?

Reply:
It's needed for some convenient stats functions in module `AbstractWalkers`. `Statistics` is a base Julia package, shouldn't add any load to the performance.